### ISSUE TITLE
reproducable build environment

### ISSUE THREAD

**Issue by erictapen** [FinnIckler/Functional-Programming-WS1516#2]
Hey Finn, da ich wieder in der Vorlesung sitze: Kónntest du mir einmal aufschreiben, mit welchem Command du das PDF generierst? Dann schreibe ich noch eine Makefile und eine nix-expression, damit der Zusammenhang zwischen .tex-Datei und .pdf-Datei deterministisch wird. Dann muss ich mich nciht mehr darum kümmern, dass mit meinen Verbesserungen immer eine gute PDF mitkommt, weil die Texenvironment in der Repo selbst definiert ist.

**Comment by FinnIckler** [comment]
Klar. Das einzige Problem is nur ich das die .hs Dateien nicht Veröffentlichen darf, da Professor Grust das nicht will. Die musst du also selber in deinen Ordner legen. Ich kann sie dir auch in einer Mail schicken wenn ihr noch nicht Zugriff auf alle habt. 

Aber sonst ist das ganz einfach. Compilieren mit XeTeX mit pygmentize installiert.

**Comment by erictapen** [comment]
In erictapen/Functional-Programming-WS1516@fc2571decf042d98269bd490d54806e8ed1e2658 is now a working version of a `nix` build environment with documentation. I'll overlook the formatting in the PDF file again and will file a PR then.

**Comment by erictapen** [comment]
In erictapen/Functional-Programming-WS1516@fc2571decf042d98269bd490d54806e8ed1e2658 is now a working version of a `nix` build environment with documentation. I'll overlook the formatting in the PDF file again and will file a PR then.

**Comment by erictapen** [comment]
fixed in #3.
